plugin/fuzz/source/dextool/plugin/fuzz/backend/fuzzvariant.d

```diff
@@ -626,45 +627,52 @@ import cpptooling.data.type;
 			    string min = minmax["min"];
 			    string max = minmax["max"];
 			    string type_type = minmax["type"];
-                string type_ns = minmax["namespace"];
+			    string type_ns = minmax["namespace"];
 			    
 			    final switch (type_type) {
```

Comment:
final has to meaning here. might as well remove it.
It is useful in those cases one switch over an enum time to guarantee at compile time that all enum literals are handled.